examples/integrations/cdi/pokemons/src/main/resources/META-INF/init_script.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO POKEMONTYPE VALUES (1, 'Normal');
```

Comment:
Just a note: If you're using H2, remember to futz with the JDBC URL in the right way (can't remember what that is at the moment) such that closing all the connections in the pool doesn't nuke your in-memory database, rendering this DDL useless.

Reply:
See @tomas-langer's comment, we may not be able to use H2 :(